webapp/src/main/resources/db/migration/V65__Alter_pull_run_text_variants.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE pull_run_text_unit_variant
+DROP FOREIGN KEY FK__PULL_RUN_TEXT_UNIT_VARIANT__PULL_RUN_ASSET_ID;
```

Comment:
Do we need to drop the 2 FKs? isn't it just re-recreating them?

Reply:
If I remember correctly I needed to drop the keys to be able to drop the index, the V61 migration has similar drop and recreate commands.